src/api/article/article.model.tests.js

```diff
@@ -55,15 +74,16 @@ describe('Article model', () => {
   it('should fail to save video article | no horizontal image', async () => {
     const errorHandler = spy(({ message }) => {
       expect(message).to.not.empty();
-      expect(message.images.horizontal).to.includes('required');
+      expect(message.images.horizontal.type).to.includes('required');
```

Comment:
just fyi, `to includes` sounds weird. I guess the intention of developers was that we either use `to include` or `includes`